metricbeat/module/kubernetes/state_container/data.go

```diff
@@ -0,0 +1,89 @@
+package state_container
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/module/kubernetes/util"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+const (
+	// Nanocores conversion 10^9
+	nanocores = 1000000000
+)
+
+func eventMapping(families []*dto.MetricFamily) ([]common.MapStr, error) {
```

Comment:
I would kind of hope instead of adding dependency on prometheus here, that this can "depend" on the prometheus module / metricset.